reference/configuration/framework.rst

```diff
@@ -477,47 +890,218 @@ would be ``/images/logo.png?version=5``.
+
+form
+....
+
+resources
+"""""""""
+
+**type**: ``string[]`` **default**: ``['FrameworkBundle:Form']``
+
+A list of all resources for form theming in PHP. This setting is not required
+if you're using the Twig format for your templates, in that case refer to
+:ref:`the form book chapter <book-forms-theming-twig>`.
+
+Assume you have custom global form themes in
+``src/WebsiteBundle/Resources/views/Form``, you can configure this like:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/config.yml
+        framework:
+            templating:
+                form:
+                    resources:
+                        - 'WebsiteBundle:Form'
+
+    .. code-block:: xml
+
+        <!-- app/config/config.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:framework="http://symfony.com/schema/dic/symfony"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/symfony http://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+            <framework:config>
+
+                <framework:templating>
+
+                    <framework:form>
+
+                        <framework:resource>WebsiteBundle:Form</framework:resource>
+
+                    </framework:form>
+
+                </framework:templating>
+
+            </framework:config>
+        </container>
+
+    .. code-block:: php
+
+        // app/config/config.php
+        $container->loadFromExtension('framework', array(
+            'templating' => array(
+                'form' => array(
+                    'resources' => array(
+                        'WebsiteBundle:Form'
+                    ),
+                ),
+            ),
+        ));
+
+.. note::
+
+    The default form templates from ``FrameworkBundle:Form`` will always be
+    included in the form resources.
+
+.. seealso::
+
+    See :ref:`book-forms-theming-global` for more information.
+
+.. _reference-templating-base-urls:
+
+assets_base_urls
+................
+
+**type**: ``{ http: [], ssl: [] }``
+
+This option allows you to define base URLs to be used for assets referenced
+from ``http`` and ``ssl`` (``https``) pages. A string value may be provided in
+lieu of a single-element array. If multiple base URLs are provided, Symfony2
+will select one from the collection each time it generates an asset's path.
+
+For your convenience, ``assets_base_urls`` can be set directly with a string or
+array of strings, which will be automatically organized into collections of base
+URLs for ``http`` and ``https`` requests. If a URL starts with ``https://`` or
+is `protocol-relative`_ (i.e. starts with ``//``), it will be added to both
+collections. URLs starting with ``http://`` will only be added to the
+``http`` collection.
```

Comment:
There's a lot of talking here, and we mention twice that you can pass an array of strings or just a string. I think we should explain less, and show 2 different code blocks instead:

1) Show an example without using the `http` and `ssl` keys - use the "array" format, but maybe a comment that you could just pass a string. Use protocol-relative URLs (because these are really what you should be using), but maybe show a commented example of an `https://` example

2) Show an example using `http` and `ssl`, probably using the array format again

My 2 cents on this - basically, show some code! :)